docs/api/checkouts.rst

```diff
@@ -238,185 +373,291 @@ Response will contain an updated order information.
+            "last_name":"Gregory",
+            "country_code":"US",
+            "street":"300 E St SW",
+            "city":"\u2019Washington",
+            "postcode":"DC 20546"
+        },
+        "billing_address":{
+            "first_name":"Frederick D.",
+            "last_name":"Gregory",
+            "country_code":"US",
+            "street":"300 E St SW",
+            "city":"\u2019Washington",
+            "postcode":"DC 20546"
+        },
+        "payments":[
+            {
+                "id":21,
+                "method":{
+                    "id":1,
+                    "code":"cash_on_delivery"
                 },
-                "order": {
-                    "href": "/app_dev.php/api/v1/orders/52"
+                "amount":103549,
+                "state":"cart"
+            }
+        ],
+        "shipments":[
+            {
+                "id":21,
+                "state":"cart",
+                "method":{
+                    "code":"dhl_express",
+                    "enabled":true
                 }
-            },
-            "id": 2,
-            "amount": 1504750,
-            "created_at": "2014-12-15T14:57:28+0000",
-            "updated_at": "2014-12-15T14:57:28+0000",
-            "state": "new"
-        }
+            }
+        ],
+        "currency_code":"USD",
+        "locale_code":"en_US",
+        "checkout_state":"shipping_selected"
     }
 
+Payment step
+------------
 
-With that information, another PUT request with the id of payment method is enough to proceed:
+When we are done with shipping choices and we know the final price of an order, we can select a payment method.
+
+Definition
+^^^^^^^^^^
 
 .. code-block:: text
 
-    PUT /api/v1/checkouts/44
+    GET /api/v1/checkouts/select-payment/{id}
+
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| id            | url attribute  | Id of the requested cart             |
++---------------+----------------+--------------------------------------+
 
-Parameters
-~~~~~~~~~~
+.. warning::
 
-payments[X][method]
-    The id of the payment method, where X is the payment number.
-    Leave empty to add new
+    Similar to the shipping step, this one has its own controller, which has to be replaced if you want to make some changes. Exemplary implementation can be found `here <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ApiBundle/Controller/ShowAvailablePaymentMethodsController.php>`__
 
-Response
-~~~~~~~~
+Example
+^^^^^^^
 
-Response will contain the updated order information.
+To check available payment methods for the cart that has a shipment methods assigned, we need to execute this curl command:
```

Comment:
To check what payment methods are available for the cart that has a **shipment/shipping method** assigned

Choose one ;>